Mathlib/RingTheory/Polynomial/Quotient.lean

```diff
@@ -143,6 +143,34 @@ theorem polynomialQuotientEquivQuotientPolynomial_map_mk (I : Ideal R) (f : R[X]
   rw [RingEquiv.symm_apply_apply, polynomialQuotientEquivQuotientPolynomial_symm_mk]
 #align ideal.polynomial_quotient_equiv_quotient_polynomial_map_mk Ideal.polynomialQuotientEquivQuotientPolynomial_map_mk
 
+/-- Let `P` be a polynomial over `R`. If `P` is a unit, then all its coefficients are nilpotent,
+except its constant term which is a unit. -/
+theorem _root_.Polynomial.IsUnit.coeff_isUnit_isNilpotent {P : Polynomial R} (hunit : IsUnit P) :
```

Comment:
It feels strange to have the result in this file. I guess this is because this is the first files that imports everything you need?

Reply:
My instinct would be to put it in `RingTheory.Polynomial.Basic`. I've asked about it (and added some details) on [Zulip](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Reintroducing.20quotient.20rings.20in.20RingTheory.2EPolynomial.2EBasic).

Reply:
Thank you!

Reply:
Okay, we got Johan's approval to import `Mathlib.RingTheory.Ideal.Quotient` in `RingTheory.Polynomial.Basic`, so you can just move everything there.

Reply:
The file which I actually need is `Mathlib.RingTheory.Ideal.QuotientOperations` (I also need `Ideal.mk_ker` from that file. I should have mentioned it earlier but I had forgotten about it. I apologize for this.). Is it fine to import `Mathlib.RingTheory.Ideal.QuotientOperations` in `RingTheory.Polynomial.Basic`?

Reply:
Yes that's fine (this is precisely the import that was removed during the port).

Reply:
That's great news. I moved the results from `RingTheory.Polynomial.Quotient` to `RingTheory.Polynomial.Basic`. I have also changed the names of those results for more clarity.